### PR TITLE
Add default value for baseColorFactor in tile-converter

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**

### CODE REVIEW

**Comment by dryabinin-actionengine** [comment]
@Tamrat-B  Fix for error as you mentioned
image

**Comment by dryabinin-actionengine** [comment]
> Is there an easy way to add a test case?

Unfortunately not. To add a test case here we need to add it for all **geometry-converter** file.

File: modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js [hunk 1]
```diff
@@ -483,9 +483,12 @@ function convertMaterial20(sourceMaterial, sourceImages) {
...
     };
   } else {
-    material.pbrMetallicRoughness.baseColorFactor = sourceMaterial.pbrMetallicRoughness.baseColorFactor.map(
-      c => Math.round(c * 255)
-    );
+    // Should use default baseColorFactor if it is not present in source material
+    // https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#reference-pbrmetallicroughness
+    const defaultBaseColorFactor = [1, 1, 1, 1];
+    const baseColorFactor =
+      sourceMaterial.pbrMetallicRoughness.baseColorFactor || defaultBaseColorFactor;
```
[the remark this answers is not in the document]
Comment (dryabinin-actionengine) [reply modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js:490]: Fixed

**Comment by ibgreen** [comment]
You could create unit tests for the helper functions.

**Pull request merged**